### PR TITLE
Add Serp

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

**Review by PyExplorer — APPROVED** [review]

File: zyte_common_items/pages/serp.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+from .base import BasePage, Page
+from .mixins import HasMetadata
+
+
+class BaseSerpPage(BasePage, Returns[Serp], HasMetadata[SerpMetadata]):
+    pass
+
+
+class SerpPage(Page, Returns[Serp], HasMetadata[SerpMetadata]):
```
Comment (PyExplorer) [inline zyte_common_items/pages/serp.py:18]: Just wondering, how and where will `SerpPage` be used?

Comment (Gallaecio) [reply zyte_common_items/pages/serp.py:18]: It should work as a base class for custom page objects for Serp items.

**Review by BurnzZ — APPROVED** [review]
LGTM!

**Comment by VMRuiz** [comment]
The fields that defined so far looks good. However, some other important fields like subtitlet, or media type - videos, images - are missing. Are you implementing only the essential fields at this point and leaving the rest of feature revisions?

File: zyte_common_items/components/__init__.py [hunk 2]
```diff
@@ -21,4 +21,5 @@
 from .ratings import AggregateRating, StarRating
 from .real_estate import RealEstateArea
 from .request import Header, ProbabilityRequest, Request
+from .serp import SerpOrganicResult
```
Comment (kmike) [inline zyte_common_items/components/__init__.py:24]: Currently for other listing items list elements are not in the components module, they're in items directly - see e.g. product list or article list.

**Pull request merged**